sovrin_client/test/cli/test_pool_upgrade_validation.py

```diff
@@ -0,0 +1,24 @@
+import pytest
+
+from sovrin_node.test.upgrade.conftest import validUpgrade
+from sovrin_client.test.cli.constants import INVALID_SYNTAX
+from plenum.common.util import randomString
+from sovrin_common.constants import JUSTIFICATION
+
+
+def testPoolUpgradeHasInvalidSyntaxIfJustificationIsEmpty(be, do, validUpgrade, trusteeCli):
+    validUpgrade[JUSTIFICATION] = ''
```

Comment:
I get it but doesn't it sound odd? If i don't want to give a justification, i will not include it in txn. So i was asking what is the expectation? Is it mandatory or not?